pkg/services/accesscontrol/migrator/zanzana.go

```diff
@@ -111,9 +114,9 @@ func managedPermissionsCollector(store db.DB) TupleCollector {
 		for _, p := range permissions {
 			var subject string
 			if len(p.UserUID) > 0 {
-				subject = zanzana.NewObject(zanzana.TypeUser, p.UserUID)
+				subject = zanzana.NewObject(zanzana.TypeUser, p.UserUID, "")
 			} else if len(p.TeamUID) > 0 {
-				subject = zanzana.NewObject(zanzana.TypeTeam, p.TeamUID)
+				subject = zanzana.NewObject(zanzana.TypeTeam, p.TeamUID, "member")
```

Comment:
maybe we could make a `NewSubject` construct and not a `NewObject`.

Reply:
I don't agree but maybe we can have a better name like new tuple key? both object and subject/user is built from a tuple key

Reply:
Ah then I agree to renaming to `NewTupleKey` or `NewTuple`